config/spotbugs-exclude.xml

```diff
@@ -45,6 +45,46 @@
       <Method name="main" />
       <Bug pattern="DM_EXIT" />
     </Match>
+    <Match>
+        <Class name="com.puppycrawl.tools.checkstyle.Main" />
+        <Method name="loadProperties" />
+        <Bug pattern="RCN_REDUNDANT_NULLCHECK_OF_NONNULL_VALUE" />
```

Comment:
Each suppress have corresponding comment to explain reason, please provide.
If case is complicated, it is better to do wide explanation in issue comment and make xml comment ` see link for details `

Reply:
it would speed up process, if you start replyinig as "done" for each review process.